docs/layers/grid-layer.md

```diff
@@ -15,28 +15,24 @@ Inherits from all [Base Layer](/docs/layers/base-layer.md) properties.
 
 ## Layer-specific Properties
 
-##### `latDelta` (Number, optional)
+##### `latOffset` (Number, optional)
 
 - Default: `0.0089`
 
 Latitude increment of each cell
 
-##### `lngDelta` (Number, optional)
+##### `lonOffset` (Number, optional)
 
 - Default: `0.0113`
 
 Longitude increment of each cell
 
-##### `enable3d` (Boolean, optional)
+##### `extruded` (Boolean, optional)
 
 - Default: `true`
 
 Whether to enable grid elevation. If se to false, all grid will be flat.
 
-##### `opacity` (Number, optional)
-
-- Default: `0.6`
-
```

Comment:
Just realized that there may be another decision point here - is it better to have elevation as a separate accessor rather than as z coord of position (like in line layer)?